File: Java/3005.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        int[] freq = new int[101];
        int max = Integer.MIN_VALUE;
        for (int num : nums) {
            freq[num]++;
            if (freq[num] > max) {
                max = freq[num];
            }
        }
        int result = 0;
        for (int i : freq) {
            if (i == max) {
                result += max;
            }
        }
        return result;
    }
}

File: Python/3005.py
from typing import Counter, List


class Solution:
    def maxFrequencyElements(self, nums: List[int]) -> int:
        freq = Counter(nums)
        max_freq = max(f for f in freq.values())
        return sum(max_freq if f == max_freq else 0 for f in freq.values())
